pandas/tests/strings/test_strings.py

```diff
@@ -599,7 +599,7 @@ def test_normalize_index():
     ],
 )
 def test_index_str_accessor_visibility(values, inferred_type, index_or_series):
-    from pandas.core.strings import StringMethods
+    from pandas.core.strings.accessor import StringMethods
```

Comment:
any reason for this not to be at the top of the file?

Reply:
Don't think so. Moved to the top